app/graphql/mutations/create_order_form.rb

```diff
@@ -20,7 +20,30 @@ def resolve(customer:, email:, comment:, total:, products:)
 
       OrderFormMailer.confirmation_email(customer, email, comment, total, products).deliver_now
 
+      send_data_to_google_sheets(order_form)
+
       { message: "Hello #{customer}, your order submission was successful! An order request confirmation will be sent to #{email} shortly. If you don't receive that email please reach out to us at contact@everuseproducts.com"}
     end
+
+    private
+
+    def send_data_to_google_sheets(order_form)
+      service_account_path = Rails.root.join('app/graphql/mutations/sheets.json')
+      service = GoogleSheetsService.new(service_account_path)
+    
+      products_array = JSON.parse(order_form.products)
+      products_string = products_array.map do |product|
+        product.map { |key, value| "#{key.capitalize}: #{value}" }.join(", ")
+      end.join(" | ")
+
+      values = [
+        [order_form.customer, order_form.email, order_form.comment, order_form.total, products_string]
+      ]
```

Comment:
does order_form have a created/updated at value here since it was saved to the db? if so can we pull the date into the values/columns of this spreadsheet

Reply:
I added in the code for formatting the date and adding in a column and value here!